### DISCUSSION TITLE
Is JReleaser suitable to simplify releases to MavenCentral?

### DISCUSSION THREAD

**Discussion by sdelamo** [jreleaser/jreleaser#872] [General]
I have been checking JReleaser documentation, and it seems to target the release of executable distributions. Is JReleaser also suitable to simplify releases of JARs to MavenCentral?

**Comment by aalmiray** [comment]
No, JReleaser does not post JARs & POMs to Maven Central, yet. This is because both Maven and Gradle already support this feature as part of their normal lifecycles. However, in the spirit of the tool being a release focused one it might makes sense for it to also support this option in the future.

**Reply by sdelamo** [reply]
Is the tool already compatible for maven/gradle builds which already define the Maven Publication using standard  publishing plugins. 

E.g. https://github.com/sdelamo/eurorates/blob/master/build.gradle

**Reply by aalmiray** [reply]
It is. The release process is decoupled from the build process and publication to Maven Central.

**Reply by aalmiray** [reply]
FWIW I use the Kordamp Gradle Plugins suite to simplify project configuration, artifact signing, publication to Maven Central, and other bits

**Comment by aalmiray** [comment]
This is now possible with the upcoming `1.3.0` https://github.com/jreleaser/jreleaser/issues/953
